blosc2/blosc2_ext.pyx

```diff
@@ -498,11 +540,34 @@ def get_blocksize():
     """
     return blosc1_get_blocksize()
 
+cdef _check_cparams(blosc2_cparams *cparams):
+    if cparams.nthreads > 1:
+        if BLOSC2_USER_REGISTERED_CODECS_START <= cparams.compcode <= BLOSC2_USER_REGISTERED_CODECS_STOP:
+            raise ValueError("Cannot use multi-threading with user defined codecs")
```

Comment:
This could be supported in the future, could you replace "Cannot use" -> "Cannot use (yet)"